File: mario.c
#include <stdio.h>
#include <cs50.h>

void print_row(int length);


int main(void)
{
    int height = get_int("Height: ");
    for (int i = 0; i < height; i++)
    {
        print_row(height);


    }
}

void print_row(int length)
{
    for (int j = 0; j < length; j++)
        {
            printf("#");
        }
    printf("\n");
}

File: 50x/sentimental-mario-more/mario.py
from cs50 import get_int
x = 0
while True:
    x = get_int("Height:")

    if x < 9 and x > 0:
        break
    else:
        print("Usage: int 1-8")

for i in range(x):
    line = str((" " * (x - i-1)) + ("#" * (i+1)) + "  " + ("#" * (i+1)))
    print(line)
